src/ParametricOptInterface.jl

```diff
@@ -816,6 +816,20 @@ function MOI.is_valid(
     return MOI.is_valid(model.optimizer, c)
 end
 
+function MOI.is_valid(
```

Comment:
@joaquimg I am not sure if this is the correct thing to do here or just pass directly to the `model.optimizer` I guess this is more important for `ScalarQuadratics` that might be parametrized.

Reply:
`is_valid` just means: do you know what this is?

It's probably sufficient to say `haskey(model.affine_constraint_cache, c) || MOI.is_valid(model.optimizer, c)`

Reply:
But not all the affine expressions will be in the cache just the parametrized ones.

Reply:
Even if it's parametrized, `add_constraint` always return the index of the inner optimizer so I agree with @jd-lara, we could just return `MOI.is_valid(model.optimizer, c)`.

Reply:
you have to convert index before passing to inner due to quadratic to affine conversion

Reply:
@joaquimg yeah if we keep `param*var` then it is still necessary to have a different dispatch for `is_valid` that checks the appropriate cache.